package.json

```diff
@@ -117,13 +117,13 @@
     "sass-loader": "10.1.0",
     "stylelint": "13.7.2",
     "stylelint-scss": "3.18.0",
-    "turbo": "^1.2.9",
+    "turbo": "1.6.0",
```

Comment:
Any higher than this seems to trigger the threat detection on my machine, I'm raising this with infosec.